File: pset6/mario/mario.py
# Makes stairs like the one at the end of World 1-1 in Super Mario Bros.

from cs50 import get_int

while True:
    h = get_int("Height:")
    if h >= 0 and h <= 23:
        break

for i in range(h):

    for j in range(h - i - 1):
        print(" ", end="")

    for k in range(i + 2):
        print("#", end="")

    print("")

File: pset1/mario/less/mario.c
// Makes stairs like the one at the end of World 1-1 in Super Mario Bros.
#include <stdio.h>
#include <cs50.h>



int main(void)
{
    int h;

    do
    {
        h = get_int("Height:");
    }
    while (h < 0 || h > 23);

    for (int i = 0; i < h; i++)
    {
        // Making spaces before stairs
        for (int j = 0; j < h - (i + 1); j++)
        {
            printf(" ");
        }
        // Making Hashes for stairs
        for (int k = 0; k < i + 2; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}
